ompi/attribute/attribute.c

```diff
@@ -393,25 +391,30 @@ do { \
  * Cases for attribute values
  */
 typedef enum ompi_attribute_translate_t {
+    OMPI_ATTRIBUTE_INVALID,
```

Comment:
Doesn't this break internal ABI? Maybe do ```OMPI_ATTRIBUTE_INVALID = -1``` to avoid shifting the existing values by one.

Reply:
or move it at the end as OMPI_ATTRIBUTE_MAX_VALID.

Reply:
There is no internal ABI, it's all in the same file. Changed it to -1 nevertheless.